File: articles/the-original-grit-scale-and-what-it-measures/cpp/original_grit_scale_score.cpp
#include <iostream>

double grit_score(double perseverance_effort, double consistency_interest) {
    return 0.60 * perseverance_effort + 0.40 * consistency_interest;
}

int main() {
    std::cout << "Synthetic Original Grit Scale score: "
              << grit_score(0.77, 0.63)
              << std::endl;
    return 0;
}

File: articles/the-original-grit-scale-and-what-it-measures/fortran/original_grit_scale_score.f90
program original_grit_scale_score
  implicit none
  real :: perseverance_effort, consistency_interest, grit_score

  perseverance_effort = 0.77
  consistency_interest = 0.63
  grit_score = 0.60 * perseverance_effort + 0.40 * consistency_interest

  print *, "Synthetic Original Grit Scale score:", grit_score
end program original_grit_scale_score
